packages/documentation-spa/docs/apps/hosted/research-repository/research-repository-spa.md

```diff
@@ -91,17 +89,17 @@ Analyst Papers Microservice provides a set of GraphQL Queries and Mutation APIs
 
 ## FAQs
 
-* **Where is data/documents in the Analyst Papers coming from?**
+* **Where is data/documents in the Research Repository coming from?**
 
-    The Analyst Papers app has custom integrations with PNT Portal, and Google Drive which allows it to fetch and search for Analyst Papers documents.
+    The Research Repository app has custom integrations with PNT Portal, and Google Drive which allows it to fetch and search for Analyst Papers documents.
 
     For PNT Portal, it searches the documents from the ‘Analyst Materials’ table. And for Google Drive, it performs the search in the "Researches in One Platform" shared folder.
 
 * **How frequently is the data on the Homepage updated?**
 
-    The Homepage shows the documents from a cache stored in the Analyst Papers backend. The cache is updated every day to check for new document additions/removals.
+    The Homepage shows the documents from a cache stored in the Research Repository backend. The cache is updated every day to check for new document additions/removals.
 
     But unlike the homepage, newly added documents are searchable almost immediately. So they show up in search even though it doesn’t show up on the homepage.
 
-* **Analyst Papers Architecture**
-    ![img](./analyst-papers-architecture.svg)
+* **Research Repository Architecture**
+    ![img](./research-repository-architecture.svg)
```

Comment:
Can you also change the "Analyst Papers Cache" to "Research Papers Cache" in the architecture diagram?

The editable diagram is in the Google Doc. Just rename it and save it as an image.

Reply:
Good catch!